keylime/tenant.py

```diff
@@ -235,6 +245,34 @@ def init_add(self, args):
             # Process allowlists
             self.allowlist = ima.process_allowlists(al_data, excl_data)
 
+        # Read command-line path string TPM2 event log template
+        if "mb_refstate" in args and args["mb_refstate"] is not None:
+            _policy_pcrs = list(self.tpm_policy.keys())
+            _policy_pcrs.remove('mask')
+            _protected_pcrs = config.MEASUREDBOOT_PCRS
+
+            for _pcr in _policy_pcrs :
```

Comment:
We're doing this check twice (line 209 above). I know it's in different methods, but just wanted to check that we need to have it in both places. And if we do, maybe it should be in it's own helper method?